spec.md

```diff
@@ -603,6 +603,19 @@ interface Literal <: Node, Expression {
 
 A literal token. Note that a literal can be an expression.
 
+### RegexLiteral
+
+```js
+interface RegexLiteral <: Literal {
+  regex: {
+    pattern: string;
+    flags: string;
+  };
+}
+```
+
+The `regex` property allows regexes to be represented in environments that don’t
```

Comment:
This is really helpful.
On Feb 17, 2015 12:53 PM, "Ingvar Stepanyan" <notifications@github.com>
wrote:

> In spec.md <https://github.com/estree/estree/pull/29#discussion_r24836592>
> :
>
> > @@ -603,6 +603,19 @@ interface Literal <: Node, Expression {
> >
> >  A literal token. Note that a literal can be an expression.
> >
> > +### RegexLiteral
> > +
> > +```js
> > +interface RegexLiteral <: Literal {
> > +  regex: {
> > +    pattern: string;
> > +    flags: string;
> > +  };
> > +}
> > +```
> > +
> > +The `regex` property allows regexes to be represented in environments that don’t
>
> Yeah, I guess so.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/estree/estree/pull/29/files#r24836592>.
>